atf_eregs/settings/dev.py

```diff
@@ -3,6 +3,18 @@
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
 
+# Analytics settings
+ANALYTICS = {
+    'GOOGLE': {
+       'GTM_SITE_ID': '',
+       'GA_SITE_ID': '',
+    },
+    'DAP': {
+        'AGENCY': 'DOJ',
+        'SUBAGENCY': 'ATF',
+    },
```

Comment:
If we don't want to send GA information, we probably don't want to send DAP info either, hey?